SUMMARY.md

```diff
@@ -8,10 +8,9 @@
 * [Get the code, build and run](get-started/build-tidb-from-source.md)
 * [Setup an IDE](get-started/setup-an-ide.md)
 * [Write and run unit tests](get-started/write-and-run-unit-tests.md)
-* [Commit code and submit a pull request](get-started/commit-code-and-submit-a-pull-request.md)
+* [Using Git](get-started/using-git.md)
```

Comment:
I prefer to keep the original title, it forms a step in the whole journey

Reply:
I'm ok to put all content first.

Reply:
done.